<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterWindow</class>
 <widget class="QDialog" name="RegisterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Sign Up</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>160</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="labelRUsername">
      <property name="text">
       <string>Username :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelRPassword">
      <property name="text">
       <string>Password :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelRConfirmPass">
      <property name="text">
       <string>Confirm Password :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelRAge">
      <property name="text">
       <string>Age</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelGender">
      <property name="text">
       <string>Gender</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelAddress">
      <property name="text">
       <string>Address</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>50</y>
     <width>160</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLineEdit" name="lineEditRUsername"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditRPassword">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditRConfirmpassword">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditRAge"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditRGender">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditRAdd"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>320</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
